docs/glossary.md

```diff
@@ -0,0 +1,168 @@
+
+## D
+- [Delegate Call](#delegate-call)
+
+## E
+- [EOA](#eoa)
+
+## H
+- [Hollow Accounts](#hollow-accounts)
+
+## L
+- [Leaky Tests](#leaky-tests)
+
+## P
+- [Precompiles](#precompiles)
+
+## S
+- [Security Model V2](#security-model-v2)
+- [Sidecars](#sidecars)
+- [Synthetic Transaction](#synthetic-transactions)
+- [System Contracts](#system-contracts)
+- [System Transaction](#system-transaction)
+
+## T
+- [Token Allowance](#token-allowance)
+
+## W
+- [Weibar](#weibar)
+
+---
+
+## Address Book
+**Address Book**: The _**network**_ address book is data structure that contains information (such as node ID, alias, IP address, and weight) for all nodes in the network, 
+not just consensus, but also block nodes, future validator nodes and more.
+The address book is stored as a file on disk at the moment but will be stored purely in state as part of Dynamic Address Book phase 3.	Disambiguation: There is also an "address book" in files 0.0.101 and 0.0.102 which is different to the "network address book" which are manually managed by devops.
+
+
+## Account Aliases
+**Account Aliases**: These are identifiers which are public key values, and are mostly 33 byte EVM address values. The "triplet" form (e.g. 0.0.18273892) is the simpler and easier to "type".
+An alias is not intended to be user-friendly, but instead to offer a smart-contract friendly identifier. See https://hips.hedera.com/hip/hip-32 and https://hips.hedera.com/hip/hip-583 for more information.
+
+
+## Admin Transaction 
+**Admin Transaction**: A HAPI transaction which has been signed by the council, for example a Freeze Transaction
+
+
+## Application Accounts
+**Application Accounts:** Used by developers and users interacting with applications deployed on Hedera. These accounts enable users to interact with the functionality provided by dApps, such as gaming, finance, or other decentralized services.
+
+## Auto Associations
+**Auto Associations**: The ability of accounts to automatically associate with tokens. An auto association is one or more slots you approve that allow tokens to be sent to your contract or account without explicit authorization for each token type. If this property is not set, you must approve each token manually (via the `TokenAssociateTransaction` in the SDKs) before it can be received, held, or sent by that account.
+https://hips.hedera.com/hip/hip-904 - currently in progress, will represent a significant change which allows for the number of "automatic association" slots to be set to -1 which represents infinite slots.
+
+
+## Auto-Renew Accounts
+**Auto-Renew Accounts**: A feature designed to automatically renew the lifecycle of certain entities (like accounts, files, smart contracts, topics, or tokens) by funding their renewal fees. This feature is particularly useful for entities that require continuous operation over long periods, as it automates the process of keeping these entities active by periodically charging a linked auto-renew account. 
```

Comment:
> A feature designed to automatically renew the lifecycle of certain entities
If we are describing feature we should probably just say `AutoRenewal`

Reply:
Thanks @Neeharika-Sompalli 🎉 Fixed it